### ISSUE TITLE
Replicated results not matching paper results

### ISSUE THREAD

**Issue by NyleSiddiqui** [Gpn-star/SCNet#1]
Hello authors, 

Thanks for releasing your code publicly. I have ran your code exactly as is on the PRCC dataset with your processed masks and am only able to attain a max top-1 of 46.5 on the CC protocol, compared to the 61.3 reported in the paper. Moreover, I even changed the weight for the VC_loss to 0.1 according to the paper, since it was hard-coded to be 0.01. Similarly, I am only able to attain a top-1 of 29.6 on LTCC compared to 47.5 reported in the paper. This leads me to believe this is not a dataset specific problem.

Could you please upload your pre-trained weights somewhere, or clone this repo and run the code to see if there may be any errors? Of course this may be on my end as well, but I have extensively reviewed the implementation and am not able to find the problem. Thanks in advance for your help!

**Comment by jiangliman** [comment]
me too, i have reimplemented the baseline method in Table 3, the Rank-1 and mAP on the PRCC can only reach 37, compared to the 46.3 and 43.2 in the paper. I have rechecked the code carefully, and i believe it only use the Raw info.
Have you deal with it？What is the bug in it. 
Thank you!

**Comment by Gpn-star** [comment]
Thanks for your question. Our code is based on distributed training. Changing the number of parallel GPUs for model training to 2 or more may solve your problem.
